src/test/java/org/takes/http/BkParallelTest.java

```diff
@@ -80,12 +83,16 @@ public Response act(final Request req)
             }
         };
         new Thread(
+            // @checkstyle AnonInnerLengthCheck (23 lines)
             new Runnable() {
                 @Override
                 public void run() {
                     try {
                         new FtBasic(
-                            new BkParallel(new BkBasic(take)),
+                            new BkParallel(
+                                new BkBasic(take),
+                                count
```

Comment:
@lautarobock so to solve the problem we need **more** threads, not less. Your code makes **less** threads than three.